uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/input-with-copy-button.html

```diff
@@ -0,0 +1,11 @@
+<div class="input-with-copy-button-component row">
```

Comment:
Hm... I am not sure we need a directive here, as commented earlier, not sure it is flexible enough. Maybe we simply use 

```js
input.select(); // not sure how do pass the input though...
$document[0].execCommand('copy');
input.blur();
```

Reply:
As discussed some time before with @AkA84 components should not access the DOM, that's why the the directive was implemented.